stdlib/universe/moving_average.go

```diff
@@ -338,49 +378,60 @@ func (t *movingAverageTransformation) doUInt(vs *array.Uint64, b execute.TableBu
 					return err
 				}
 			}
-			next := window[int64(i+1)%t.n]
+
+			next := t.window[bj][int64(t.i[bj]+1)%t.n]
 			if next != nil {
-				sum -= next.(uint64)
-				count--
+				*sum -= next.(uint64)
```

Comment:
i'm a little surprised that this works since `next` is technically a values.Value  type, right?

Reply:
`next` is either int64, uint64. or float64:
`func (a *Int64) Value(i int) int64    { return a.values[i] }`